extension/src/test/e2e/util.ts

```diff
@@ -193,11 +231,19 @@ export const expectAllPlotsToBeFilled = async (webview: PlotsWebview) => {
 }
 
 export const findScmTreeItems = async () => {
-  const workspace = await browser.getWorkbench()
-  const activityBar = workspace.getActivityBar()
+  const workbench = await browser.getWorkbench()
+  const activityBar = workbench.getActivityBar()
   const sourceControlIcon = await activityBar.getViewControl('Source Control')
 
   await sourceControlIcon?.openView()
 
+  const visibleItems = await findCurrentTreeItems()
+
+  await visibleItems[visibleItems.length - 1].click()
+
+  for (let i = 0; i < 20; i++) {
+    await browser.keys('ArrowDown')
```

Comment:
[F] Because we now have more tree items (https://github.com/iterative/dvc/issues/9602) the items towards the bottom of the tree were not shown and because they weren't shown they weren't in the document (at all!). We need to make them visible to be able to assert against them which is why we need this code.